activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Introduce config to allow all queries to be retried.
+
+    Active Record config `retry_queries` can be set to enable retries across all database queries,
+    according to the retries configuration in the application's database config. The value will
+    default to false (retries disabled).
+
+    *Adrianna Chang and Scott Yang*
```

Comment:
I think since we don't want to document this option we shouldn't have a changelog entry.